internal/services/appservice/windows_web_app_resource.go

```diff
@@ -123,6 +125,13 @@ func (r WindowsWebAppResource) Arguments() map[string]*pluginsdk.Schema {
 
 		"identity": helpers.IdentitySchema(),
 
+		"key_vault_reference_identity_id": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ValidateFunc: msivalidate.UserAssignedIdentityID,
```

Comment:
double-checking this - is this the Identity used to access the Key Vault? in which case this name is misleading

Reply:
Yes, and it's the same as the resources being superseded.  I agree it could be misleading, but couldn't think of a better name and figured it would be safest to keep what users are used to and reflective of the docs? More than happy to change it if you can offer a suggestion?